### PR TITLE
test: add i18n to cli-hello-world integration test

### PR DESCRIPTION
**Pull request by ocombe**
Adding a test for i18n locales to the cli hello world integration test

### CODE REVIEW

**Review by filipesilva — APPROVED** [review]

**Comment by alexeagle** [comment]
I think we also need an update to `integration/README.md` which currently says that we upgrade the CLI by throwing away this app and generating it fresh. We had no local modifications previously so there was no reason to be careful about upgrading.
I think the right text there now should read that we use `ng update` command to keep this example app up-to-date as CLI changes things?

**Comment by ocombe** [comment]
There's a few stuff that needs to be updated for sure, run_test.sh tests for "hello_world__render3__cli" which doesn't exist anymore afaik. Same thing for _payload-limits.json with "hello_world__render3__rollup" and "hello_world__render3__cli" (unless they are generated somehow?).
But it should probably a different PR?

**Comment by alexeagle** [comment]
IMO this PR breaks the README so that should be fixed here - the other
tests I agree can be separate, I don't really understand the plan there.

File: integration/cli-hello-world/package.json [hunk 1]
```diff
@@ -21,27 +25,29 @@
...
+    "concurrently": "3.4.0",
     "core-js": "file:../../node_modules/core-js",
+    "lite-server": "2.2.2",
     "rxjs": "file:../../node_modules/rxjs",
     "zone.js": "file:../../node_modules/zone.js"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.5.0",
-    "@angular/cli": "^6.0.0-rc.0",
+    "@angular/cli": "^6.0.0-rc.5",
```
Comment (alexeagle) [inline integration/cli-hello-world/package.json]: hmm we need some plan for doing this upgrade more frequently...
not in this PR :)

File: integration/cli-hello-world/src/app/app.module.ts [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
-
+import { LOCALE_ID, NgModule } from '@angular/core';
+import { registerLocaleData } from '@angular/common';
+import localeFr from '@angular/common/locales/fr';
 
 import { AppComponent } from './app.component';
 
+registerLocaleData(localeFr);
```
Comment (alexeagle) [inline integration/cli-hello-world/src/app/app.module.ts]: include a comment pointing to the issue so we know why this is here - otherwise seems unneeded

**Review by alexeagle — APPROVED** [review]
would be nice to update that README too

File: integration/cli-hello-world/package.json (hunk 1 above)
Comment (ocombe) [reply integration/cli-hello-world/package.json]: the update is necessary in this PR because the fix is in rc5

**Pull request closed without merging**